src/memjs/memjs.ts

```diff
@@ -74,7 +82,23 @@ export type GivenClientOptions<Value, Extras> = Partial<BaseClientOptions> &
     SerializerProp<Value, Extras>
   >;
 
-class Client<Value, Extras> {
+export type CASToken = Buffer;
+
+export interface GetResult<Value = MaybeBuffer, Extras = MaybeBuffer> {
+  value: Value;
+  extras: Extras;
+  cas: CASToken | undefined;
+}
+
+export type GetMultiResult<
+  Keys extends string = string,
+  Value = MaybeBuffer,
+  Extras = MaybeBuffer
+> = {
+  [K in Keys]: GetResult<Value, Extras>;
+};
+
+class Client<Value = MaybeBuffer, Extras = MaybeBuffer> {
```

Comment:
is all this really for tests? I should suck it up and make them just be all buffers